extension/src/js/background/index.js

```diff
@@ -120,6 +153,17 @@ chrome.tabs.onRemoved.addListener(function(tabId, removed) {
     }
 });
 
+function addTitleListener() {
+    chrome.tabs.onUpdated.addListener(function(tabId, changeInfo) {
+        if(changeInfo.title && tabId === tabID) {
```

Comment:
Line #158 in background/index.js, tabId and tabID is confusing to me, is there a way we can have a different var name?